test/commands/verify.spec.js

```diff
@@ -0,0 +1,60 @@
+// Import Node.js Dependencies
+import dotenv from "dotenv";
+import { spawn } from "child_process";
+import { fileURLToPath } from "url";
+import path from "path";
+
+dotenv.config();
+
+// Import Third-party Dependencies
+import test from "tape";
+import splitByLine from "split2";
+
+// CONSTANTS
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const kProcessDir = path.join(__dirname, "..", "process");
+
+const expectedLines = `directory size:     2 KB
```

Comment:
Could it be better to store those expected output in seperated files? (instead of having them like that in the code).